### PR TITLE
Add BizNet mainnet

### PR DESCRIPTION
**Pull request by TrustHenry**
This is our site. : https://bosagora.io

### CODE REVIEW

File: _data/chains/eip155-2151.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "BOSAGORA BizNet",
+  "chain": "ETH",
+  "network": "mainnet",
```
Comment (ashutoshpw) [inline _data/chains/eip155-2151.json:4]:
```suggestion
```

Comment (ashutoshpw) [reply _data/chains/eip155-2151.json:4]: Please remove the `network` param as of our latest docs and also run this command locally to format the JSON properly.

`npx prettier --write --ignore-unknown _data`

**Review by ashutoshpw — CHANGES_REQUESTED** [review]

**Comment by ashutoshpw** [comment]
@TrustHenry You need to run `npx prettier --write --ignore-unknown _data` from the project directory to format the JSON file properly and pass the Workflow `
Check JavaScript for conformance with Prettie`

**Comment by TrustHenry** [comment]
@ashutoshpw Thankyou. 
I ran `npx prettier --write --ignore-unknown _data`
Could you run the workflow again?

**Review by ashutoshpw — APPROVED** [review]
Looks good to me. @ligi can merge it in the `main` branch

**Comment by TrustHenry** [comment]
@ligi Please a review.

**Comment by TrustHenry** [comment]
#1809 We start with a new name.(BOSagora)

**Pull request closed without merging**